.github/workflows/build-and-analyze.yml

```
@@ -48,6 +48,9 @@ jobs:
         run: |
           chmod +x dbsetup.sh
           ./dbsetup.sh
+      - name: Restart database to enable config changes
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2046:warning:1:16: Quote this to prevent word splitting

(shellcheck)